docs/src/modules/components/AppNavDrawer.js

```diff
@@ -54,17 +54,28 @@ PersistScroll.propTypes = {
 };
 
 // https://github.com/philipwalton/flexbugs#3-min-height-on-a-flex-container-wont-apply-to-its-flex-items
-const ToolbarIE11 = styled('div')({ display: 'flex' });
+const ToolbarIE11 = styled('div')({
+  display: 'flex',
+  position: 'sticky',
+  top: 0,
+  zIndex: 1050,
+  backgroundColor: 'inherit',
+});
 
 const ToolbarDiv = styled('div')(({ theme }) => {
   return {
     ...theme.mixins.toolbar,
     paddingLeft: theme.spacing(3),
+    height: '65px',
     display: 'flex',
     flexGrow: 1,
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'space-between',
+    backgroundColor: theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : '#fff',
+    borderBottom: '1px solid',
+    borderColor:
+      theme.palette.mode === 'dark' ? theme.palette.primaryDark[700] : theme.palette.grey[100],
```

Comment:
This file also should not change? @hbjORbj

Reply:
These are changes by @danilo-leal to bring the algolia to the most front relative to other components (e.g., side nav).

Reply:
I'll resolve this comment so we centralize discussions around it on the above one since they both have the same questioning!